corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -378,6 +398,27 @@ def _with_progress(self, doc_types, iterable, progress_name='Migrating'):
             self.log_info("{} ({})".format(doc_count, ', '.join(doc_types)))
             return iterable
 
+    def _log_objects_processed_count(self, tags, throttled=False):
+        if throttled and self.processed_objects % 100 != 0:
+            return
+
+        datadog_counter("commcare.couchsqlmigration.forms_processed",
```

Comment:
rename to `objects_processed` or `docs_processed` since it's not always forms?

Reply:
oops, i'd meant to do that. nice catch